.github/workflows/test.yml

```diff
@@ -4,9 +4,6 @@ on:
   - pull_request
   - push
 
-env:
-  BUNDLE_WITHOUT: release
-
```

Comment:
This can stay, but you should clean it up in the release workflow.

Reply:
done